build.gradle

```diff
@@ -18,7 +18,7 @@ plugins {
     id 'com.jfrog.bintray' version '1.8.4'
 }
 
-version = '2.10.9'
+version = '2.10.10'
```

Comment:
since you are adding a new endpoint, I would say we should increase the MINOR version (https://semver.org/) and have 2.11.0 instead?

Reply:
correct